stanford/playbooks/insights.yml

```diff
@@ -1,16 +1,39 @@
-- name: Deploy Analytics Insights
-  hosts: ~tag_Name_{{ machine }}_{{ env }}
-  become: True
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/cluster.yml
+  vars:
+    CLUSTER_NAME: insights
+  when: redeploy | default(False) | bool
+
+- name: Deploy insights service
+  hosts: "tag_Cluster_{{ CLUSTER_NAME }}:&tag_todo_deploy:&tag_Deployment_{{ COMMON_DEPLOYMENT }}:&tag_Number_{{ CLUSTER_NUMBER }}"
   gather_facts: True
-  vars_files:
-    - "{{ COMMON_DEPLOYMENT_DIR }}/insights.yml"
+  become: True
   vars:
-    COMMON_ENABLE_DATADOG: False
-    COMMON_ENABLE_SPLUNKFORWARDER: False
+    CLUSTER_NAME: insights
+    migrate_db: "yes"
+    disable_edx_services: false
+    ENABLE_DATADOG: False
+    ENABLE_SPLUNKFORWARDER: False
+    ENABLE_NEWRELIC: False
```

Comment:
These three `ENABLE_*` variables already default to `False`, so we should omit them here.